app/src/main/java/pl/szczodrzynski/edziennik/data/api/edziennik/github/ContributorsData.kt

```diff
@@ -0,0 +1,39 @@
+package pl.szczodrzynski.edziennik.data.api.edziennik.github
+
+import com.google.gson.GsonBuilder
+import okhttp3.*
+import pl.szczodrzynski.edziennik.data.api.edziennik.github.Contributor
+import pl.szczodrzynski.edziennik.App
+import java.io.IOException
+import kotlin.coroutines.suspendCoroutine
+
+object ContributorsData {
+
+    suspend fun getContributors(app: App): List<Contributor> {
+
+        val request = Request.Builder()
+            .url("https://api.github.com/repos/szkolny-eu/szkolny-android/contributors")
+            .build()
+
+        val call = app.http.newCall(request)
+
+        return suspendCoroutine { cont ->
```

Comment:
Try to move the `suspendCoroutine` to L12, so it's an expression function (the `request` and `call` can be created inside the coroutine). The explicit return type specification is not needed then. This way you can reduce the indentation by one level.